.github/workflows/npmpublish.yml

```diff
@@ -0,0 +1,105 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [published]
```

Comment:
There is a `prereleased` trigger also, so that is another route to handle the npm tagging. Just fyi.